Source/Csla/DataPortalClient/HttpProxyOptions.cs

```diff
@@ -36,6 +36,16 @@ public class HttpProxyOptions
     /// </summary>
     public Func<IServiceProvider, HttpClient> HttpClientFactory { get; set; } = DefaultHttpFactory;
 
+    /// <summary>
+    /// Gets or sets the timeout value in milliseconds for the HTTP request.
+    /// </summary>
+    public int Timeout { get; set; }
```

Comment:
As long as we're doing this for CSLA 9, let's make this into a TimeSpan - breaking change, but better code.